### ISSUE TITLE
Crouch speed?

### ISSUE THREAD

**Issue by l0m-dev** [rehlds/ReGameDLL_CS#309]
Is there a way to change how much you slow down when crouching?

**Comment by Nord1cWarr1or** [comment]
Crouch speed is client-side physics — the crouch movement values live in pm_shared (client prediction code); the game DLL only controls the `maxspeed` cap. This is outside the game DLL scope (engine/client territory). Closing as not planned.

**Comment by Nord1cWarr1or** [comment]
Yes — since #851 the gamedll has a per-player crouch speed multiplier: `CSPlayer::m_flDuckSpeedMultiplier` (`regamedll/public/regamedll/API/CSPlayer.h`). The server movement code honors it in `pm_shared` — it overrides the vanilla `PLAYER_DUCKING_MULTIPLIER` (0.333) for ducked movement (`PM_Duck`) and for ducked ladder climbing (`PM_LadderMove`, `regamedll/pm_shared/pm_shared.cpp`). Set it per player via ReAPI (`set_member(id, m_flDuckSpeedMultiplier, Float:value)`); 0 keeps vanilla behavior.
